js/app.js

```diff
@@ -0,0 +1,17 @@
+// topnav menu //
+let mainNav=document.getElementById('menu-links');
+let navbarToggle=document.getElementById('navbar-toggle');
+
+navbarToggle.addEventListener('click',function(){
+
+    if(this.classList.contains('active')){
+        mainNav.style.display="none";
+        this.classList.remove('active');
+    }
+    else{
+        mainNav.style.display="flex";
+        this.classList.add('active');
+    }
+});
```

Comment:
Fantastic job pushing yourself into JS!!! I know you have some experience, so it's even better that you're not being shy about using it! It can only get stronger with practice.